code-challenges.js

```diff
@@ -14,21 +14,50 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
+// create a function called temp that takes in a number called degree.
+// const temp = (degree) => {
+// if degree is more than or equal to 213, return "(degree) is above boiling point."
+//     if(degree >= 213) {
+//     return `${degree} is above boiling point`
+// }
+// if degree is less than or equal to 211, return "(degree) is below boiling point."
+//     else if(degree <= 211) {
+//     return `${degree} is below boiling point`
+// }
+// if degree is equal to 213, return "(degree) is at boiling point."
+//     else {
+//     return `${degree} is at boiling point`
+// } 
+// }
+// console.log(temp(221))
+// console.log(temp(212))
+// console.log(temp(9))
 // --------------------2) Create a function that takes in two arrays of numbers and returns a single array with all the values sorted least to greatest.
 // Use the test variables provided below. Expected output: [-9, -7, 0, 3, 7, 8, 9, 13, 22, 36]
 
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 22, 9, 13]
 
+// create a function called myNumbers3 that combines the first two variables myNumbers1 and myNumbers2
+// const myNumbers3 = (myNumbers1.concat(myNumbers2)) => {
+// return an output that sorts the numbers from least to greatest.
+//     return myNumbers3.sort((a, b)=> a-b)
+// }
+
+
+
+
 
 
 // --------------------3) Create a function that takes in a string of a single word and returns the string with all letters reversed.
 // Use the test variables provided below. Expected output: "ovarb", "eilrahc"
 
 var myString1 = "bravo"
 var myString2 = "charlie"
+// combine .split to break up each letter into its own array then reverse the array with .reverse and finally join each letter back together to return the required output. 
+// I tried creating a function around this combination but I was unable to.
```

Comment:
What happened when you did? the logic looks good!

Reply:
I believe I fixed it after I commented that so it should be good lol.